lib/main.dart

```diff
@@ -140,7 +153,7 @@ class _HomePageState extends State<HomePage> {
         maxLines: null,
         decoration: new InputDecoration(
```

Comment:
`new` is not needed In new versions of dart. @alifarazz should handle CI for avoiding these kind of problems.

Reply:
I did not use the keyword new in my code, it was used by previous developers.

Reply:
was my mistake.
You're right.
@alifarazz should take care of this.